### PR TITLE
INTPYTHON-459 Refactor handling of client into a pytest fixture

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ It contains the following packages.
     - `MongoDBAtlasParentDocumentRetriever` - new feature, docs coming soon!
 - Indexing
     - `MongoDBRecordManager` - new feature, docs coming soon!
+- Document loader
+    - `MongoDBLoader` - new feature, docs coming soon!
 - LangGraph persistence
```
Comment (caseyclements) [inline README.md]: This feels misleading. Should we use qualified names? I'm for breaking this into sections by package.

**Review by caseyclements — CHANGES_REQUESTED** [review]
Bit of a misleading PR name for what's mainly an refactor of client to a fixture ;)  (which I totally agree with btw). Let's discuss the README.md

**Comment by blink1073** [comment]
Ah, drat, I never added the new files.

File: README.md (hunk 1 above)
Comment (blink1073) [reply README.md]: Yeah, let's do that.  I'll make this PR just about the refactor with a new ticket.  In the changelog PR we can break out the README.

**Review by caseyclements — APPROVED** [review]
lgtm

**Pull request merged**